examples/httpd/convox.yml

```diff
@@ -3,14 +3,26 @@ resources:
     type: postgres
   rediscache:
     type: redis
+  memcache:
+    type: memcached
+    options:
+      version: 1.6.17
+  mysqldb:
+    type: mysql
+    options:
+      version: 8.0.30
+  mariadb:
+    type: mariadb
```

Comment:
Do we need to test this specific version? Should we test using the latest?

Reply:
otherwise aws arm e2e fails
default version doesn't support arm architecture

Reply:
@nightfury1204 can you open an issue to update the default version?
e.g. [mariadb](https://github.com/convox/convox/blob/775fd9b00bb7683acf184fcccca5790278d60d89/provider/k8s/template/resource/mariadb.yml.tmpl)

Reply:
i.e. [mysql version](https://github.com/convox/convox/blob/ddbd5df1c6c22e357741c43664264346793b1795/provider/k8s/template/resource/mysql.yml.tmpl#L65)

Reply:
already created one for that: https://github.com/convox/issues-private/issues/621